sdcm/nemesis.py

```diff
@@ -519,8 +519,7 @@ def _add_and_init_new_cluster_node(self, old_node_ip=None, timeout=HOUR_IN_SEC *
         self.set_current_running_nemesis(node=new_node)  # prevent to run nemesis on new node when running in parallel
         new_node.replacement_node_ip = old_node_ip
         try:
-            self.cluster.wait_for_init(node_list=[new_node], timeout=timeout, wait_db_up=False, check_node_health=False)
```

Comment:
wasn't the case to set `wait_db_up=True` and fix the issue, but allow this flexibility to `wait_for_init`?

Reply:
Maybe, but I mostly afraid of this to trick us again in the future.
If the flexibility is there, someone may use it and we may be in the same problem again.